op-node/withdrawals/utils.go

```diff
@@ -172,8 +173,15 @@ func FinalizeWithdrawalParameters(ctx context.Context, l2client ProofClient, txH
 	if err != nil {
```

Comment:
Could add a log Index argument to this method to point to the withdrawal log. (and can make that index relative to other withdrawals, or to all log events).

Reply:
I could update this function to return a `[]FinalizedWithdrawalParameters` but didn't because we don't really need that right now, will do it if you think I should

Reply:
it's probably fine, not too difficult to do later once we actually need it